test/test-loop-close.c

```diff
@@ -55,3 +55,21 @@ TEST_IMPL(loop_close) {
 
   return 0;
 }
+
+static void loop_instant_close_work_cb(uv_work_t* req) {
+}
+
+static void loop_instant_close_after_work_cb(uv_work_t* req, int status) {
+}
+
+TEST_IMPL(loop_instant_close) {
+  uv_loop_t loop;
+  uv_work_t req;
+  ASSERT(0 == uv_loop_init(&loop));
+  ASSERT(0 == uv_queue_work(&loop,
+                            &req,
+                            loop_instant_close_work_cb,
+                            loop_instant_close_after_work_cb));
+  MAKE_VALGRIND_HAPPY();
+  return 0;
```

Comment:
This test as is cannot work: `loop` and `req` go out of scope (and get clobbered) but they're still in use.